### PR TITLE
Patients Table

### PR DESCRIPTION
**Pull request by Johngtka**
obraz

### CODE REVIEW

**Comment by Johngtka** [comment]
obraz

**Comment by Johngtka** [comment]
squash and merge proposal:
feat: Patients table with data getting from mockoon

File: src/app/app.component.css [hunk 1]
```diff
@@ -31,6 +31,12 @@ mat-divider {
   margin: 16px;
 }
 
+@media screen and (max-width: 400px) {
+  .content {
+    margin: 0;
+  }
+}
+
```
Comment (macrog) [inline src/app/app.component.css:39]: what is this for?

File: src/app/patient-table/patient-table.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core'
+import { PatientService } from './../services/patient-service'
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.ts:2]: separate imports by empty line please

**Review by macrog — COMMENTED** [review]
just a couple of things

File: src/app/app.component.css (hunk 1 above)
Comment (Johngtka) [reply src/app/app.component.css:39]: this is the style that defines no margins at a maximum screen width of 400

File: src/app/patient-table/patient-table.component.ts (hunk 2 above)
Comment (Johngtka) [reply src/app/patient-table/patient-table.component.ts:2]: done

File: src/app/patient-table/patient-table.component.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+@Component({
+  selector: 'app-patient-table',
+  templateUrl: './patient-table.component.html',
+  styleUrls: ['./patient-table.component.css'],
+})
+export class PatientTableComponent {
+  dataSource!: any[]
+  constructor(private patientService: PatientService) {}
+  ngOnInit(): void {
+    this.patientService.getPatients().subscribe((patients) => {
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.ts:12]: please add Type for patients -> patients: Array<Patient>

File: src/app/patient-table/patient-table.component.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      this.dataSource = patients
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.ts:13]: to be able to catch error when mockoon in not connected please use this structure to display error in snackbar

{
            next: (data: Array<Patient>) => (this.dataSource = patients),
            error: (err) => {
              this.snackService.showSnackBarMessage(
                'ERROR.PATIENT_TABLE_GET_PATIENTS',
                SNACK_TYPE.error,
              )
              console.log(err.message)
            },
          }

**Review by macrog — CHANGES_REQUESTED** [review]
add this please

**Review by macrog — APPROVED** [review]
looks good 👏

**Pull request merged**